src/constants.js

```diff
@@ -166,3 +166,278 @@ export const MAIN_MARKETPLACE_CONTRACT = MARKETPLACE_CONTRACT_TEIA // the one th
 export const MAIN_MARKETPLACE_CONTRACT_SWAP_TYPE = SWAP_TYPE_TEIA
 
 export const BURN_ADDRESS = 'tz1burnburnburnburnburnburnburjAYjjX'
+
+export const COVER_COMPRESSOR_OPTIONS = {
+  quality: 0.85,
+  maxWidth: 1024,
+  maxHeight: 1024,
+}
+
+export const THUMBNAIL_COMPRESSOR_OPTIONS = {
+  quality: 0.85,
+  maxWidth: 350,
+  maxHeight: 350,
+}
+
+export const LICENSE_TYPES = {
+  none: 'None',
+  'cc-by-4.0': 'CC-BY-4.0 (Attribution)',
+  'cc-by-sa-4.0': 'CC BY-SA 4.0 (Attribution ShareAlike)',
+  'cc-by-nd-4.0': 'CC BY-ND 4.0 (Attribution-NoDerivs)',
+  'cc-by-nc-4.0': 'CC BY-NC 4.0 (Attribution-NonCommercial)',
+  'cc-by-nc-sa-4.0': 'CC BY-NC-SA 4.0 (Attribution-NonCommercial-ShareAlike)',
+  'cc-by-nc-nd-4.0': 'CC BY-NC-ND 4.0 (Attribution-NonCommercial-NoDerivs)',
+  custom: 'Custom (Specify)',
+}
+
+export const LANGUAGES = {
```

Comment:
@NoRulesJustFeels Do you have a better source 😅 . 
I'm not sure Country ISO code is what was in the metadata proposal, but I improvised just to fill it for now

Reply:
This is the standard for the language based on the TZIP-21 spec: https://datatracker.ietf.org/doc/html/rfc1766
But it seems there have been other standards that have replaced this. Based on a search, I found this list: https://www.loc.gov/standards/iso639-2/php/code_list.php